index.html

```diff
@@ -75,6 +75,14 @@
         <div class="status" id="sync-idle-users-update"></div>
       </div>
 
+      <div class="button-container">
+        <button class="action-button" id="sync-idle-7d-Plus-users">
+          <span class="spinner"></span>
+          Sync Idle 7d+ Users On Discord
+        </button>
+        <div class="status" id="sync-idle-7d-Plus-users-update"></div>
```

Comment:
why `div` tag is used instead of `p` tag?

Reply:
if you look into code every buttons which are present in `index.html` is  following same format like they are using div for showing the status as you can see in below snippet this is another button
```suggestion
        <div class="button-container">
            <button class="action-button" id="sync-nicknames">
                <span class="spinner"></span>
                 Sync Users Nicknames
            </button>
        <div class="status" id="sync-nicknames-status-update"></div>
      </div>
```